Tapawingo_backend/Interface/IRoutepartsRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Tapawingo_backend.Models;
+
+namespace Tapawingo_backend.Interface
+{
+    public interface IRoutepartsRepository
+    {
+        Task<Routepart> CreateRoutePart(Routepart newRoutepart);
```

Comment:
in IFileRepository en IDestinationRepository noem je async functies bijvoorbeeld SafeFileAsync terwijl je hier geen Async in de functienaam toevoegd.